src/App.tsx

```diff
@@ -1,24 +1,267 @@
-/* eslint-disable max-len */
```

Comment:
Remove unused eslint-disable comments at lines 1 and 2. The linter should not report any issues related to `max-len` and `jsx-a11y/control-has-associated-label`.